app/views/dashboard_v2/_topicCard.html.erb

```diff
@@ -0,0 +1,91 @@
+<%# Most of this design has been extracted from the tags _topicCard with slight variations %>
+<% subscriptions ||= @tag_subscriptions  # allow overriding with local variable %>
+<div class="card-view pt-5">
+  <% subscriptions.each do |subscription| %>
+    <div class="card shadow-sm p-2 mb-3 bg-white rounded" >
+      <div class="card-header mt-2" style=" padding:0.8em; background-color: inherit; border:none;">
+      <%# Unfollow section %>
+        <% if current_user && current_user.following(subscription.tag.name) %>
```

Comment:
@RuthNjeri Whoaaa great work :tada: :clap:  If possible you may check for n+1 query problem in `subscription.tag.name` or other such references where we might be connecting two tables and looping over. Thiss looks amazing!

Reply:
Thanks for the review @Tlazypanda 😄  .. I think the N + 1 query is avoided from the controller, I included the tag model in the subscription query.  I'll be sure to keep an eye out for any references connecting two tables...

https://github.com/publiclab/plots2/blob/59a5daa6212c40f05c72ce3bffa64e3fdf94eadd/app/controllers/home_controller.rb#L57